NachoSpigot-Server/src/main/java/me/rastrian/dev/threads/AsyncThread.java

```diff
@@ -0,0 +1,94 @@
+package me.rastrian.dev.threads;
+
+import io.netty.channel.Channel;
+import io.netty.util.concurrent.Future;
+import io.netty.util.concurrent.GenericFutureListener;
+import me.elier.nachospigot.config.NachoConfig;
+import me.rastrian.dev.threads.netty.Spigot404Write;
+
+import java.util.Queue;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import net.minecraft.server.NetworkManager;
+import net.minecraft.server.Packet;
+import net.openhft.affinity.AffinityLock;
+import net.openhft.affinity.AffinityStrategies;
+
+public abstract class AsyncThread {
+    private boolean running = true;
+    private int TICK_TIME = 1000000000 / NachoConfig.combatThreadTPS;
```

Comment:
Final static here?

Reply:
Won't this cause issues? Since the `init` method in NachoConfig sets this variable. If this variable initializes before NachoConfig is initialized this will break.

Reply:
At my tests it's working fine.

Reply:
Same here 👍